### DISCUSSION TITLE
更新Mac 15.2系统后，切换拼音方案快捷键失效，菜单栏无法使用（如图），有没有大神知道如何解决呢？先谢过了。

### DISCUSSION THREAD

**Discussion by James-zhao-9** [rime/home#1768] [Q&A] [answered]
更新Mac 15.2系统后，切换拼音方案快捷键失效，菜单栏无法使用（如图），有没有大神知道如何解决呢？先谢过了。
![Screenshot 2024-12-28 at 12 20 50 PM](https://github.com/user-attachments/assets/dadb9691-7a1e-4573-b122-38ba0e8376f4)

**Comment by LEOYoon-Tsaw** [comment]
截圖顯示輸入法掛了，請描述如何復現

**Reply by James-zhao-9** [reply]
感谢您的回复，快捷键切换的问题已解决，原来是外接键盘的问题（更新固件后，快捷键切换方案问题解决），但是如上图菜单栏问题（更新15.2后出现）依然存在。是不是权限设置的问题（如下面图片所示，图标为灰色）。
![Screenshot 2024-12-29 at 9 06 49 AM](https://github.com/user-attachments/assets/a064f210-3c21-4451-96f2-ea51b30fac8f)
![Screenshot 2024-12-29 at 9 07 16 AM](https://github.com/user-attachments/assets/32e19df5-be39-4656-9140-8df3dac65b5c)
![Screenshot 2024-12-29 at 9 07 51 AM](https://github.com/user-attachments/assets/85f11274-ae16-4077-ac70-2cd40d833e78)

**Comment by James-zhao-9** [comment]
有没有遇到同样问题得朋友？你们都是如何解决的？系统更新到15.3.1问题依然存在，删除squirrel后（清除掉input methods文件下squirrel和library/rime文件夹），重新安装，问题依然存在！

**Comment by James-zhao-9** [comment, marked as answer]
在网络上诸位朋友的帮助下，问题得以解决，与bettermouse冲突，关了后者，就一切正常了

**Comment by James-zhao-9** [comment]
在网络上诸位朋友的帮助下，问题得以解决，与bettermouse冲突，关了后者，就一切正常了
